ui/src/app/machines/views/MachineList/MachineListHeader/MachineListHeader.test.tsx

```diff
@@ -240,50 +242,4 @@ describe("MachineListHeader", () => {
       wrapper.find('Button[data-test="add-hardware-dropdown"]').length
     ).toBe(0);
   });
-
-  it("can add the selected filter when displaying a form", () => {
```

Comment:
Is this going to get covered in the selected actions update?

Reply:
Yeah I missed that I'd removed those ones, I've got them in a different component in my other branch.